crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -35,15 +47,56 @@ use std::{
     sync::atomic::{AtomicI64, Ordering},
 };
 
-/// Lightweight unique ID of an entity.
+/// Lightweight identifier of an [entity](crate::entity).
+///
+/// The identifier is implemented using a generational index: a combination of an ID and a
```

Comment:
Can we add a nice link to a description of generational indexes?

Reply:
I tend to be reluctant about linking to sources that are neither official Bevy or official Rust docs. However, since it seems there is nothing to be found there, we can break that rule here.

Anyway, I barely skimmed this [medium article] some days ago.

[medium article]: https://lucassardois.medium.com/generational-indices-guide-8e3c5f7fd594

Reply:
I'm happy with this link.